ui/app/mainui/AppMain.qml

```diff
@@ -511,8 +509,30 @@ Item {
             }
         }
 
-        function onOpenSendModal(address: string) {
-            sendModal.open(address)
+        function onOpenSendModal(sendType: int, senderAddress: string,
+                                 tokenId: string, tokenType: int,
+                                 tokenAmount: string, chainId: int,
+                                 recipientAddress: string, recipientType: int,
+                                 onlyAssets: bool, interactive: bool,
+                                 publicKey: string, ensName: string,
+                                 stickersPackId: string) {
+            if (sendType === undefined) sendType = Constants.SendType.Transfer
+            if (senderAddress === undefined) senderAddress = ""
+            if (tokenId === undefined) tokenId = ""
+            if (tokenType === undefined) tokenType = Constants.TokenType.Unknown
+            if (tokenAmount === undefined) tokenAmount = ""
+            if (chainId === undefined) chainId = 0
+            if (recipientAddress === undefined) recipientAddress = ""
+            if (recipientType === undefined) recipientType = SendPopups.Helpers.RecipientAddressObjectType.Address
+            if (onlyAssets === undefined) onlyAssets = false
+            if (interactive === undefined) interactive = true
+            if (publicKey === undefined) publicKey = ""
+            if (ensName === undefined) ensName = ""
+            if (stickersPackId === undefined) stickersPackId = ""
```

Comment:
... and this will become much easier because you will have all the defaults already in